src/Extensions.Hosting.AsyncInitialization/Hosting/AsyncInitializationHostExtensions.cs

```diff
@@ -71,28 +71,28 @@ public static async Task TeardownAsync(this IHost host, CancellationToken cancel
         /// <param name="cancellationToken">Optionally propagates notifications that the operation should be cancelled</param>
         /// <remarks>
         /// Cancelling the <paramref name="cancellationToken"/> will not affect the teardown process. 
-        /// Teardown, when configured, is always performed, regardless if the process is cancelled or if an exception is thrown.
-        /// To prevent deadlocks during teardown, a <see cref="CancellationToken"/> with a <see cref="DefaultTeardownTimeout"/> value is passed to any registered initializers that implement <see cref="IAsyncTeardown"/>. 
+        /// Teardown, when configured, is always performed, even if the process is cancelled or an exception is thrown.
+        /// To prevent teardown from blocking forever, a <see cref="CancellationToken"/> with a <see cref="DefaultTeardownTimeout"/> value is passed to any registered initializers that implement <see cref="IAsyncTeardown"/>. 
         /// The entire teardown process will be cancelled if the timeout expires before all initializers have completed teardown.
         /// </remarks>
         /// <returns>The <see cref="Task"/> that represents the asynchronous operation.</returns>
         /// <exception cref="ArgumentNullException">Thrown when the host is null.</exception>
         /// <exception cref="InvalidOperationException">Thrown when the initialization service has not been registered.</exception>
         /// <exception cref="OperationCanceledException">Thrown when the cancellationToken is cancelled.</exception>
         /// <exception cref="TimeoutException">Thrown when teardown times out.</exception>
+        /// <exception cref="AggregateException">Thrown when multiple exceptions occur.</exception>
         public static async Task InitAndRunAsync(this IHost host, CancellationToken cancellationToken = default) 
             => await host.InitAndRunAsync(DefaultTeardownTimeout, cancellationToken).ConfigureAwait(false);
 
 
-        /// <param name="host">The <see cref="IHost"/> to initialize and run.</param>
+#pragma warning disable 1573
+        /// <inheritdoc  cref="InitAndRunAsync(IHost, CancellationToken)" path="/*[not(self::remarks)]"/>
         /// <param name="teardownTimeout">The <see cref="TimeSpan"/> timeout value to use for teardown. Setting this value to <see cref="Timeout.InfiniteTimeSpan"/> will disable timeout handling.</param>
-        /// <param name="cancellationToken">Optionally propagates notifications that the operation should be cancelled</param>
         /// <remarks>
         /// Cancelling the <paramref name="cancellationToken"/> will not affect the teardown process. 
         /// Teardown, when configured, is always performed, regardless if the process is cancelled or if an exception is thrown, using a timeout as specified by the <paramref name="teardownTimeout"/> parameter.
```

Comment:
Same remark as on the other overload

Reply:
Not sure if I get what you mean.
The remarks are not identical.

Reply:
Sorry, I meant I had the same comment about the wording as for the `<remark>` tag on the other overload. i.e.:
```suggestion
        /// Teardown, when configured, is always performed, even if the process is cancelled or an exception is thrown, using a timeout as specified by the <paramref name="teardownTimeout"/> parameter.
```